components/brave_extension/extension/brave_extension/background/reducers/shieldsPanelReducer.ts

```diff
@@ -48,6 +48,7 @@ const updateShieldsIconImage = (state: State) => {
 const updateShieldsIcon = (state: State) => {
   updateShieldsIconBadgeText(state)
   updateShieldsIconImage(state)
+  shieldsPanelState.updateThemeType(state)
```

Comment:
Hey @cezaraugusto just wondering why we're calling `updateThemeType` inside `updateShieldsIcon`. I'm not sure I see the connection between the shields icon being affected by state of the page and shield-blocking-state and the brave theme being affected by user choice.